sympy/polys/rings.py

```diff
@@ -3041,6 +3032,156 @@ def subresultants(self, g, x=None):
 
         return R
 
+    def _subs(self, subs_dict):
+        ring = self.ring
+        result_poly = ring.zero
+
+        for monom, coeff in self.iterterms():
+            new_coeff = coeff
+            new_monom = list(monom)
+
+            for i, val in subs_dict.items():
+                exp = monom[i]
+                if exp > 0:
+                    new_coeff *= val ** exp
+                new_monom[i] = 0
+
+            if new_coeff:
+                new_monom = tuple(new_monom)
+                if new_monom in result_poly:
+                    result_poly[new_monom] += new_coeff
+                    if not result_poly[new_monom]:
+                        del result_poly[new_monom]
+                else:
+                    result_poly[new_monom] = new_coeff
+
+        return result_poly
+
+    def symmetrize(self):
+        r"""
+        Rewrite *self* in terms of elementary symmetric polynomials.
+
+        Explanation
+        ===========
```

Comment:
With a method like this that has a large docstring it would be better to keep the method and docstring in the superclass and then have it call `_symmetrize` which would be the subclass implementation.